resources/usa-properties-og/server.lua

```diff
@@ -854,34 +974,41 @@ function Evict_Owners()
       end
       if GetWholeDaysFromTime(info.fee.paid_time) >= max_ownable_days then
         if info.type == "house" then
-          if info.storage.money >= info.fee.price then
-            print("** Money from property (" .. name .. ") storage was used to pay for a month of rent. **")
-            PROPERTIES[name].storage.money = PROPERTIES[name].storage.money - info.fee.price
-            PROPERTIES[name].fee.paid_time = os.time()
-            PROPERTIES[name].fee.paid = true
-            local final_time = nil
-            local today = os.date("*t", os.time())
-            PROPERTIES[name].fee.due_days = HOUSE_PAY_PERIOD_DAYS
-            final_time = {day = today.day, month = today.month + 1, year = today.year}
-            local endtime = os.time(final_time)
-            PROPERTIES[name].fee.due_time = endtime
-            PROPERTIES[name].fee.end_date = os.date("%x", endtime)
-            SavePropertyData(name)
+          local ownerPlayer = exports["essentialmode"]:getDocument("characters", PROPERTIES[name].owner.identifier)
+          if type(ownerPlayer) ~= "boolean" then
+            if ownerPlayer.bank >= info.fee.price and not info.will_leave then
+              print("** Money from bank of (" .. ownerPlayer.name.first .. " " .. ownerPlayer.name.last .. ") was used to pay for a month of rent. **")
+              ownerPlayer.bank = ownerPlayer.bank - info.fee.price
+              exports["essentialmode"]:updateDocument("characters", PROPERTIES[name].owner.identifier, ownerPlayer, true)
+              PROPERTIES[name].fee.paid_time = os.time()
+              PROPERTIES[name].fee.paid = true
+              local final_time = nil
+              local today = os.date("*t", os.time())
+              PROPERTIES[name].fee.due_days = HOUSE_PAY_PERIOD_DAYS
+              final_time = {day = today.day, month = today.month + 1, year = today.year}
```

Comment:
```suggestion
              final_time = {day = today.day, month = (today.month + 1) % 12, year = today.year}
```

should be more what we want